src/classes/HH_INamingImpl.cls

```diff
@@ -31,9 +31,9 @@
 * @author Salesforce.org
 * @date 2014
 * @group Households
-* @description HH_NameSpec class that implements template based naming for Households.
+* @description NPSPs default implementation of the HH_INaming interface used to generate household names.
 */
-public with sharing class HH_NameSpec implements HH_INaming {
+public with sharing class HH_INamingImpl implements HH_INaming {
```

Comment:
I do like the "impl" naming structure (Java-ish maybe), but it hasn't fit our general naming pattern. But, it is an implementation of an interface so it makes sense from that perspective. It's probably a good practice for us to use going forward. For example, the 3 address verification classes could also follow this pattern.